### PR TITLE
Add Java sample app content to OIE embedded auth with SDK and SIW sample app guide 2

### PR DESCRIPTION
**Pull request by vanngo-okta**
- **What's changed?** Updated Java configuration and code based on PR2312 feedback from Arvin. Added 2 use cases: embedded-widget-use-case-load and embedded-widget-use-case-sign-in.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-408182](https://oktainc.atlassian.net/browse/OKTA-408182)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-If you are setting up the sample app, use this value:
-`http://localhost:8080`. Otherwise, choose
-a URI that is used by your app.
+If you are setting up the embedded authentiation sample app, use `http://localhost:8080` as the redirect URI. Otherwise, use a redirect URI that is appropriate for your app.
+
+> **Note:** The default redirect URI, `http://localhost:8080/authorization-code/callback` is listed when you create a new app integration in the Admin Console. Do not remove this redirect URI.
```
Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md:3]: Did you verify this default redirect URI?

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/sdkforyourapp.md [hunk 2]
```diff
@@ -24,17 +22,14 @@ import com.okta.idx.sdk.api.response.TokenResponse;
...
 
-Code snippet example:
-
 ```java
 AuthenticationResponse beginResponse = idxAuthenticationWrapper.begin()
 ```
 
-See [Okta Java SDK Usage guide](https://github.com/okta/okta-idx-java#usage-guide) for more information on using the SDK.
\ No newline at end of file
+See [Okta Java SDK Usage guide](https://github.com/okta/okta-idx-java#usage-guide) for more information on using the SDK.
```
Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/sdkforyourapp.md:35]:
```suggestion
See [Okta Java SDK Usage guide](https://github.com/okta/okta-idx-java#usage-guide) for more information on SDK usage.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-  If you are using the sample app use: `http://localhost:8080`.
-  Otherwise, choose a URI that your app accepts.
+  If you are using the embedded authentication sample apps, add the following redirect URI `http://localhost:8080` (in addition to the default redirect URI `http://localhost:8080/authorization-code/callback` that is already listed).
```
Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md:1]: Please verify the redirect URI. Note that this default might change in future depending on how the backend Orgs are provisioned.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-alternate-flows/main/java/getuserprofile.md [hunk 4]
```diff
@@ -1,29 +1,27 @@
 
-Using the `tokenResponse` obtained from `IDXAuthenticationWrapper`'s `authenticate`
+Using the `TokenResponse` obtained from `IDXAuthenticationWrapper`'s `authenticate`
 method, make a request to the following endpoint:
```
Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-alternate-flows/main/java/getuserprofile.md:3]:
```suggestion
method, make a request to the `/v1/userinfo` endpoint:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md:3]: I added the default redirect URI (in addition to the `http://localhost:8080`) in the Admin Console, and ran the app with redirectURI=`http://localhost:8080`, and everything still works as before for the widget sample app.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/java/integrationsteps.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+    new OktaSignIn(config).showSignInAndRedirect(
+        { el: '#sign-in-widget' },
+        function (res) {}
+    );
+    /*]]>*/
+</script>
+```
+
+The Okta Sign-In Widget renders in the sign-in page when the Spring controller `/custom-login` is triggered.
```
Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/java/integrationsteps.md:48]:
```suggestion
The Okta Sign-In Widget renders in the sign-in page when your app's login controller is hit.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/java/integrationsteps.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+### Step 3: Run your app
+
+The final step is to run your app. If the widget and your Okta org are property
```
Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/java/integrationsteps.md:52]:
```suggestion
The final step is to run your app. If the widget and your Okta org are correctly
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/java/integrationsteps.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+```javascript
+<script src="https://global.oktacdn.com/okta-signin-widget/5.8.2/js/okta-sign-in.min.js" type="text/javascript"></script>
+<link href="https://global.oktacdn.com/okta-signin-widget/5.8.2/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
+```
+
+> **Note:** The previous snippet example uses Sign-In Widget, version `5.8.2`. Use the latest version of the Sign-In Widget found [here](https://github.com/okta/okta-signin-widget/releases/).
+
+### Step 2: Add JavaScript to initialize and load the widget
+
+Load the widget in the sign-in page, similar to the following snippet:
```
Comment (arvindkrishnakumar-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/java/integrationsteps.md:16]:
```suggestion
Initialize the widget in the sign-in page, similar to the following snippet:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md:3]:
```suggestion
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md:1]:
```suggestion
  If you are using the embedded authentication sample apps, add `http://localhost:8080` as the redirect URI.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md:3]: looks good.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md (hunk 3 above)
Comment (arvindkrishnakumar-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md:1]: looks good.

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/java/redirecturi.md:1]: Discussed with Arvin: decided to remove the text in brackets.

**Pull request merged**